contentcuration/contentcuration/frontend/channelEdit/components/StudioTree/StudioTree.vue

```diff
@@ -58,9 +70,10 @@
               >
                 <template #activator="{ on }">
                   <IconButton
-                    icon="optionsVertical"
+                    icon="optionsHorizontal"
```

Comment:
I believe that @nucleogenesis has changed it to `optionsVertical` deliberately. You might need to sync on this together, so we don't end up updating it over and over again.

Reply:
ok, I'll revert for now, but we'll want to make sure this is up to spec to the designs